dynect/src/main/java/denominator/dynect/DynECTErrorDecoder.java

```diff
@@ -76,6 +76,9 @@ public Exception decode(String methodKey, Response response) {
         for (Message message : messages) {
           if ("token: This session already has a job running".equals(message.info())) {
             return new RetryableException(messages.toString(), cause, null);
+          } else if ("login: Bad or expired credentials".equals(message.info)) {
```

Comment:
possible to add a test here? https://github.com/Netflix/denominator/blob/master/dynect/src/test/java/denominator/dynect/DynECTProviderDynamicUpdateMockTest.java